### ISSUE TITLE
Does not compile against nginx 1.10.0

### ISSUE THREAD

**Issue by TJC** [flant/nginx-http-rdns#6]
```
cc -c -g -O2 -fPIE -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -g -O2 -fPIE -fst
ack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2  -I src/core -I src/
event -I src/event/modules -I src/os/unix -I /usr/include/libxml2 -I objs -I src/http -I src/http/modules -I src/
http/v2 -I src/mail -I src/stream \
                -o objs/addon/nginx-http-rdns/ngx_http_rdns_module.o \
                /opt/nginx-rdns/nginx-http-rdns/ngx_http_rdns_module.c
/opt/nginx-rdns/nginx-http-rdns/ngx_http_rdns_module.c: In function ‘merge_loc_conf’:
/opt/nginx-rdns/nginx-http-rdns/ngx_http_rdns_module.c:217:89: error: ‘ngx_resolver_t’ has no member named ‘udp_c
onnections’
     if (conf->conf.enabled && ((core_loc_cf->resolver == NULL) || (core_loc_cf->resolver->udp_connections.nelts
== 0))) {
                                                                                         ^
make[3]: *** [objs/addon/nginx-http-rdns/ngx_http_rdns_module.o] Error 1
```

**Comment by TJC** [comment]
I think there's a pull request that tries to fix this issue -- but it has a typo in it!

**Comment by distorhead** [comment]
Fixed in 733af3f9158c97cd7d52b7ad46d801158f17ac4c.
